src/server/queue_func.c

```diff
@@ -401,8 +402,14 @@ queuestart_action(attribute *pattr, void *pobject, int actmode)
 			}
 
 			/* if scheduling = True, notify scheduler to start */
-			if (server.sv_attr[SRV_ATR_scheduling].at_val.at_long)
-				set_scheduler_flag(SCH_SCHEDULE_STARTQ);
+			if (server.sv_attr[SRV_ATR_scheduling].at_val.at_long) {
+				if (find_assoc_sched_pq(pque, &psched))
+					set_scheduler_flag(SCH_SCHEDULE_STARTQ, psched);
+				else {
+					sprintf(log_buffer, "No scheduler associated with the queue %s", pque->qu_qs.qu_name);
```

Comment:
Since schedulers are now associated to partitions, please mention partition in the message (with partition name)

Reply:
Done